### PR TITLE
fix(docs): add dotfiles module reference

### PR DESCRIPTION
**Pull request by doodzik**

### CODE REVIEW

File: docs/dotfiles.md [hunk 1]
```diff
@@ -11,6 +11,21 @@ explains how it loads your repo.
 
 You can read more on dotfiles best practices [here](https://dotfiles.github.io).
 
+## Module
+
+Coder's [dotfiles module](https://registry.coder.com/modules/dotfiles)
+abstracts Coder's dotfiles command into a module. This will prompt the 
+user for their dotfiles repository URL on template creation using a
```
Comment (matifali) [inline docs/dotfiles.md:18]:
```suggestion
user for their dotfiles repository URL on workspace creation using a 
```

**Review by matifali — APPROVED** [review]

**Comment by matifali** [comment]
Thank you for the contribution.

**Comment by matifali** [comment]
Self note: backport to #14421.

**Review by matifali — APPROVED** [review]
Thank you for your contribution.

**Comment by matifali** [comment]
Hi @doodzik  can you also run `make fmt/prettier`? 

I can do it for you and push to the branch.

**Review by stirby — APPROVED** [review]

**Comment by matifali** [comment]
Hi @doodzik I can't push to your PR branch. Can you run `make fmt`? Thank you

**Pull request merged**